db/migrations/20200404_CreateTable.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS movie_info(
```

Comment:
Table name could just be `movies`.
And the column `movie_uuid` could just be `uuid` or `id`, since it is inside `movies` table, it is implicit.

Once the migration file is run, the migration file won't run again.
So please create a new migration file to rename the table or column if needed.

Reply:
I'll create separate migration file to work on your suggestion....
yeah, I get to know that concept of golang-migration by maintaining the state in 'schema_migrations' table...

Reply:
you can see the [changes](https://github.com/ashwinspg/explore-golang/pull/3/commits/ea57a0321f0bf3cf9f95343a60e59f655d9a092b) here